src/components/MAPI/apps/AppInstallModal.tsx

```diff
@@ -158,6 +148,7 @@ const AppInstallModal: React.FC<
 
   const onClose = () => {
     setVisible(false);
+    setQuery('');
```

Comment:
Small bug in the app installation modal - this resets the input field for filtering the cluster list when the user closes the modal